frontend/packages/dev-console/integration-tests/features/pipelines/pipelines-secrets.feature

```diff
@@ -37,9 +41,9 @@ Examples:
 
 @regression
 Scenario Outline: Add secret to pipeline with authentication type as SSH Key : P-011-TC04   
```

Comment:
```suggestion
Scenario: Add secret to pipeline with authentication type as SSH Key: P-011-TC04   
```

Reply:
When Examples are there, Scenario Outline is required